layers/+tools/google-calendar/README.org

```diff
@@ -0,0 +1,166 @@
+#+END_SRC
+
+You need to specify which calendar is going to be synchronized to which file. The variable =org-gcal-file-alist= expect a list where each element is an associated list containing a Calendar-ID and which file to synchronize to, [[org-gcal-cals][see example]]. If you want to synchronize multiple calendars its recommended to synchronize them with different files. This also enables the user to create events for a specific calendar. Currently =org-gcal= do not support synchronization of multiple Google accounts.
+#+NAME: org-gcal-cals
+#+BEGIN_SRC emacs-lisp
+  (setq org-gcal-file-alist '(("your_first_calendar_id@gmail.com" . "/path/to/first_schedule_file.org")
+                              ("your_second_calendar_id@gmail.com" . "/path/to/second_schedule_file.org")))
+#+END_SRC
+
+You need to include the calendar event files in =org-agenda=, do so by adding the paths to the files to the =org-agenda-files= variable.
+#+BEGIN_SRC emacs-lisp
+  (setq org-agenda-files
+        (quote ("/path/to/first_schedule_file.org" "/path/to/second_schedule_file.org")))
+#+END_SRC
+
+You can create new calendar events from any Emacs buffer with =org-capture=. This layer provides convenient accessory functions that automatically synchronize the new event to Google calendar, [[*Schedule events with org-capture][see below]].
+
+** Configure calfw
+You can configure how the calfw calendar buffer should be displayed by altering the variable =calfw-calendar-window-setup=. When you close the calendar view this layer tries to restore the previous window configuration, if you would like to disable this behavior set =calfw-restore-windows-after-quit= to nil.
+
+Calfw lets you customize the name of the months and days by altering the following arrays.
+#+BEGIN_SRC emacs-lisp
+  ;; Month
+  (setq calendar-month-name-array
+    ["January" "February" "March"     "April"   "May"      "June"
+     "July"    "August"   "September" "October" "November" "December"])
+
+  ;; Week days
+  (setq calendar-day-name-array
+        ["Sunday" "Monday" "Tuesday" "Wednesday" "Thursday" "Friday" "Saturday"])
+#+END_SRC
+
+You can also change which day the week begins with the following variable.
+#+BEGIN_SRC emacs-lisp
+  ;; First day of the week
+  (setq calendar-week-start-day 1) ; 0:Sunday, 1:Monday
+#+END_SRC
+
+In Calfw calendar view you can capture TODOs or events. To do so you need to bind a [[http://orgmode.org/manual/Capture-templates.html][org-capture template]] to the variable =cfw:org-capture-template=. The following are an example template for creating a calendar event that works with the =google-calendar/sync-cal-after-capture= function.
+#+NAME: org-event-schedule
+#+BEGIN_SRC emacs-lisp
+     (setq cfw:org-capture-template '("s" "Scedule an event" entry
+                                      (file "/path/to/a/schedule/file.org")
+                                      "* %^{Description}\n%^{LOCATION}p\n%(cfw:org-capture-day)\n%?"))
+#+END_SRC
+
+** Configure alert
```

Comment:
I previously removed the `alert` dependency since I came to the same discovery as you. This reference in the README is just remnants from that. 

Good point with `org-agenda-to-appt`. I don't have time to work on integrate it at the moment so I dont think it should be part of this PR.